frontend/src/Containers/Recipe/index.js

```diff
@@ -1 +1,69 @@
 // TODO Create a connected component to render a fetched recipe
+import Divider from "@material-ui/core/Divider"
+import List from "@material-ui/core/List"
+import ListItem from "@material-ui/core/ListItem"
+import ListItemText from "@material-ui/core/ListItemText"
+import React, { Component } from "react"
+import { connect } from "react-redux"
+import { bindActionCreators } from "redux"
+import * as actions from "../../actions"
+import { BlueButton } from "./styles"
+
+class Recipe extends Component {
+  constructor(props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+    this.state = {
+      fetchedRecipeId: "",
+    }
+  }
+
+  handleClick() {
+    this.setState({ fetchedRecipeId: this.props.selectedRecipeId })
+    this.props.fetchRecipe(this.props.selectedRecipeId)
+  }
+
+  render() {
+    console.log(this.state.fetchedRecipeId)
+    return (
+      <>
+        {this.props.selectedRecipeId ? (
+          <>
+            <BlueButton onClick={this.handleClick}>fetch recipe</BlueButton>
+            <Divider />
+          </>
+        ) : null}
+        {this.props.selectedRecipeId === this.state.fetchedRecipeId &&
```

Comment:
Had to use internal state within Recipe to see what was the last fetched recipe vs what is currently selected in the UI.

I want to avoid displaying the recipe for something that is not selected in the UI.